src/includes/getting-started-config/python.django.mdx

```diff
@@ -1,4 +1,13 @@
-To configure the SDK, initialize it with the Django integration in your `settings.py` file:
+Initialize the Python SDK with the Django integration in your `settings.py` file. This snippet:
+- Instruments your Django integration to [monitor the performance](/platforms/javascript/performance/) of your application.
```

Comment:
It also does error monitoring. Is it necessary to mention anything at all this prominently?

Reply:
We say that immediately prior in this sentence "Once this is done, the Sentry SDK captures all unhandled exceptions." 

We keep this in the `include` as performance monitoring under development for other SDKs.